tests/core/creators/manual/resizewithcssunits.md

```diff
@@ -0,0 +1,39 @@
+@bender-tags: bug, 4.14.1, 1883
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea, table
+
+## Scenario 1
+
+1. In Width input type `20cm`.
+
+1. In Height input type `10cm`.
+
+1. Press `Resize!` button.
+
+## Scenario 2
+
+1. In Width input type `20cm`.
+
+1. In Height input type `10cm`.
+
+1. Tick `is content height` checkbox.
```

Comment:
It doesn't explain how this option affects editor size. E.g. that the new size should be applied only to the editor's content. That means for each scenario we should have different expected results.